src/app/sampleData.ts

```diff
@@ -0,0 +1,13 @@
+export var data: object = {
```

Comment:
This can also be the same type that was used in the HttpTableComponent.

Reply:
Made required changes.

Reply:
data is still labelled as object instead of the new HttpTableData type. Also, it's minor, but we should declare with const instead of var.

Reply:
Just corrected both changes.